packages/console/src/components/ApplicationIcon/index.tsx

```diff
@@ -1,18 +1,32 @@
 import type { ApplicationType } from '@logto/schemas';
 import { Theme } from '@logto/schemas';
 
-import { darkModeApplicationIconMap, lightModeApplicationIconMap } from '@/consts';
+import {
+  darkModeApplicationIconMap,
+  lightModeApplicationIconMap,
+  thirdPartyApplicationIcon,
+  thirdPartyApplicationIconDark,
+} from '@/consts';
 import useTheme from '@/hooks/use-theme';
 
 type Props = {
   type: ApplicationType;
   className?: string;
+  isThirdParty?: boolean;
```

Comment:
How about we pass the application instance in, instead of having two params: `type` and `isThirdParty`?

Reply:
multiple places use this ApplicationIcon element. So far only applications and application detail page need this isThirdParty field. Optional isThirdParty field  might limit the changes to applications and application detail page only.